src/main/java/com/blackducksoftware/integration/hub/alert/channel/manager/DistributionChannelManager.java

```diff
@@ -80,6 +80,10 @@ public String sendTestMessage(final R restModel) throws AlertException {
         return "Attempting to send a test message...";
     }
 
+    public ProjectData getTestMessageProjectData() {
```

Comment:
Is this method used anywhere? I cannot find it with changes in this pull request.

Reply:
I moved its location in the file. No functional change.